Makefile

```diff
@@ -2,7 +2,11 @@
 NUM_ADDONS=10
 NUM_THEMES=$(NUM_ADDONS)
 
-COMPOSE_PROJECT_NAME?=addonsserver
+# Get the name of the Makefile's directory for the docker container base name.
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
+current_dir := $(notdir $(patsubst %/,%,$(dir $(mkfile_path))))
```

Comment:
but while I'm at it, it would be pretty nice to rewrite this as a fabric script or something